Android/src/org/droidplanner/android/proxy/mission/item/fragments/MissionDoJumpFragment.java

```diff
@@ -0,0 +1,101 @@
+package org.droidplanner.android.proxy.mission.item.fragments;
+
+import android.content.Context;
+import android.view.View;
+
+import com.o3dr.services.android.lib.drone.mission.MissionItemType;
+import com.o3dr.services.android.lib.drone.mission.item.MissionItem;
+import com.o3dr.services.android.lib.drone.mission.item.command.DoJump;
+
+import org.droidplanner.android.DroidPlannerApp;
+import org.droidplanner.android.R;
+import org.droidplanner.android.proxy.mission.MissionProxy;
+import org.droidplanner.android.proxy.mission.item.markers.MissionItemMarkerInfo;
+import org.droidplanner.android.widgets.spinnerWheel.CardWheelHorizontalView;
+import org.droidplanner.android.widgets.spinnerWheel.adapters.NumericWheelAdapter;
+
+/**
+ * Created by Toby on 7/31/2015.
+ */
+public class MissionDoJumpFragment extends MissionDetailFragment implements CardWheelHorizontalView.OnCardWheelScrollListener<Integer> {
+
+    @Override
+    protected int getResource() {
+        return R.layout.fragment_editor_detail_do_jump;
+    }
+
+    @Override
+    public void onApiConnected() {
+        super.onApiConnected();
+
+        final View view = getView();
+        typeSpinner.setSelection(commandAdapter.getPosition(MissionItemType.DO_JUMP));
+
+        final DoJump item = (DoJump) getMissionItems().get(0);
+        final Context context = getContext();
+
+        final NumericWheelAdapter adapter = new NumericWheelAdapter(context, R.layout.wheel_text_centered, getFirstWaypoint(), getLastWaypoint(), "%d");
+        final CardWheelHorizontalView<Integer> waypointPicker = (CardWheelHorizontalView) view.findViewById(R.id
+                .waypoint_picker);
+        waypointPicker.setViewAdapter(adapter);
+        waypointPicker.addScrollListener(this);
+        waypointPicker.setCurrentValue(item.getWaypoint());
+
+        final NumericWheelAdapter repeatCountAdapter = new NumericWheelAdapter(context, R.layout.wheel_text_centered, -1,
```

Comment:
@TShapinsky the repeat count should be an `EditText` which only accepts unsigned integer values. No need to arbitrary limit the max repeat count to `2000`.
For the `infinite` repeat option, add a `Checkbox` above the `repeat count` field. i.e:
`Repeat indefinitely [ ]`

If the checkbox is enabled, the repeat count `EditText` should be disabled, and vice versa.